content/reference/plugins/gitlab_ci/index.md

```diff
@@ -164,14 +176,16 @@ print-output:
 
 - `RUN_ASSERTIONS`: The results of the run's assertions, as a JSON array.
 
-### Logs
+### Logs {#logs}
```

Comment:
this might be overkill!

Reply:
What I don't really like about the auto-generated anchor, is that changing the title breaks the link but the broken link doesn't seem to show as a build-time error... So it's a bit fragile if we ever change the section titles IMO.

Reply:
Yeah, Hugo build breaks on broken links but it skips anchors... :(

Reply:
Well I guess it can be done, but manually: https://github.com/gohugoio/hugo/issues/6863#issuecomment-1551957458